### PR TITLE
Add version history for 5.9.0

### PR DESCRIPTION
**Pull request by dgault**
For a full list of included PR's see https://github.com/openmicroscopy/bioformats/milestone/63?closed=1 and https://github.com/ome/bio-formats-documentation/milestone/2?closed=1

### CODE REVIEW

File: sphinx/about/whats-new.rst [hunk 1]
```diff
@@ -1,6 +1,66 @@
...
+* added support to data repo test suite for unconfigured tests
+
+Documentation improvements:
+* fixed unstable links flagged by automated link checking
+* begun adding testing for breakages to memo files
+* clarified ordering expectation in ``getUsedFiles`` Javadocs
+* added documentation for dimensionOrder in bfsave with the :doc:`MATLAB toolbox </developers/matlab-dev>`
+* fixed broken links in previous release notes
+* expanded documentation for command-line utilities to cover undocumented 
+  :doc:`options </users/comlinetools/display>` and :doc:`environment variables </users/comlinetools/index>`
```
Comment (sbesson) [inline sphinx/about/whats-new.rst]: Travis seems to be unhappy with this line

```
Warning, treated as error:
/home/travis/build/ome/bio-formats-documentation/target/sphinx/about/whats-new.rst:60:Unexpected indentation.
[ERROR] Command execution failed.
```

File: sphinx/about/whats-new.rst [hunk 2]
```diff
@@ -1,6 +1,67 @@
 Version history
 ===============
 
+5.9.0 (2018 July 3)
+-------------------
+
+File format fixes and improvements:
+
+* MetaMorph
+   - fixed a ``NullPointerException`` when a stage label in not present
```
Comment (mtbc) [inline sphinx/about/whats-new.rst]: "is", not "in"

File: sphinx/about/whats-new.rst [hunk 3]
```diff
@@ -1,6 +1,67 @@
...
+   - ensured that reported domain is now consistent with the existence of a Plate in OME-XML
+   - fixed Metamorph RGB series channel count (thanks to Jeremy Muhlich)
+* Leica LIF
+   - improved handling of dimension order for non RGB channels
```
Comment (mtbc) [inline sphinx/about/whats-new.rst]: hyphenate non-RGB

File: sphinx/about/whats-new.rst [hunk 4]
```diff
@@ -1,6 +1,67 @@
...
+   - updated to ensure consistent ordering of used files
+
+Bug fixes and improvements:
+
+* enabled building and testing with Java 9 and 10
+* added CI testing with Java 10 on AppVeyor and Travis
+* removed Java 7 from Appveyor matrix
+* updated a number of Maven plugins to current versions
+* corrected warnings in Maven configuration in sub-components 
+* added a warning to clarify the behaviour when passing metadata with ``dimensionOrder`` in ``bfsave`` as
```
Comment (mtbc) [inline sphinx/about/whats-new.rst]: behavior

File: sphinx/about/whats-new.rst [hunk 5]
```diff
@@ -1,6 +1,67 @@
...
+* enabled building and testing with Java 9 and 10
+* added CI testing with Java 10 on AppVeyor and Travis
+* removed Java 7 from Appveyor matrix
+* updated a number of Maven plugins to current versions
+* corrected warnings in Maven configuration in sub-components 
+* added a warning to clarify the behaviour when passing metadata with ``dimensionOrder`` in ``bfsave`` as 
+  part of the Bio-Formats MATLAB toolbox (thanks to Jonathan Armond)
+* improved robustness in the detection of patterns as part of the file stitching
+* fixed a bug relating to dimension order in the Bio-Formats plugins Exporter
+* fixed download URLs in Bio-Formats command line tools
```
Comment (mtbc) [inline sphinx/about/whats-new.rst]: maybe hyphenate "command-line" to match below

File: sphinx/about/whats-new.rst [hunk 6]
```diff
@@ -1,6 +1,67 @@
...
+* added support to data repo test suite for unconfigured tests
+
+Documentation improvements:
+
+* fixed unstable links flagged by automated link checking
+* begun adding testing for breakages to memo files
+* clarified ordering expectation in ``getUsedFiles`` Javadocs
+* added documentation for dimensionOrder in bfsave with the :doc:`MATLAB toolbox </developers/matlab-dev>`
+* fixed broken links in previous release notes
+* expanded documentation for command-line utilities to cover undocumented :doc:`options </users/comlinetools/display>`
```
Comment (mtbc) [inline sphinx/about/whats-new.rst]: s/utilities/tools/ to match above, or are these different things?

**Pull request merged**